ResourceStatusSystem/Command/DowntimeCommand.py

```diff
@@ -106,6 +106,13 @@ def _prepareCommand( self ):
       if not seHost:
         return S_ERROR( 'No seHost for %s' % elementName )
       elementName = seHost
+      
+    elif elementType == 'FTS':
+    	gocdbServiceType = "FTS"
+    	try:
+    		elementName	= getFTSServers("FTS3")[ 'Value' ][0]
+    	except:
+    		return S_ERROR( 'No FTS3 server specified in dirac.cfg (see Resources/FTSEndpoints)' )
```

Comment:
I remember I wrote the getFTS3Servers method the way it is, b/c the CS template Federico suggested me to work with, was something like this (*). So the reason why I just took the first element was that the second element was FTSType. In other words I didn't know yet how you wanted to deal with multiple FTS3 servers. Now after you example I understand better (and I guess there is no FTSType attribute right?)
Besides I'm not sure to understand what you mean by "why you mangled the names?".

(*)
FTSEndpoints
{
FTS3
{
FTSEndpoint = https://fts3-pilot.cern.ch:8446
FTSType = FTS3
}
}